samples/FakeItEasy.Examples.VB/AssertingCalls.vb

```diff
@@ -13,4 +13,11 @@ Public Class AssertingCalls
 
         A.CallTo(Function() factory.CreateWithColor(A(Of String).Ignored)).MustHaveHappened()
     End Sub
+
+    Public Sub Asserting_ordered_function_calls()
+        Dim factory = A.Fake(Of IWidgetFactory)()
+
+        A.CallTo(Function() factory.CreateWithColor(A(Of String).Ignored)).MustHaveHappened().Then(

```

Comment:
A nitpick, and I'm not sure what's idiomatic in VB, but I would expect the line break to be before `.Then(...`